wicket/src/wicketd.rs

```diff
@@ -116,6 +135,35 @@ impl WicketdManager {
         });
     }
 
+    fn start_ignition_command(
+        &self,
+        component_id: ComponentId,
+        command: IgnitionCommand,
+        poll_inventory_now: mpsc::Sender<()>,
+    ) {
+        let log = self.log.clone();
+        let addr = self.wicketd_addr;
+        tokio::spawn(async move {
+            let client = create_wicketd_client(&log, addr, WICKETD_TIMEOUT);
+            let sp: SpIdentifier = component_id.into();
+            let res =
+                client.post_ignition_command(sp.type_, sp.slot, command).await;
+            // We don't return errors or success values, as there's nobody to
+            // return them to. How do we relay this result to the user?
+            slog::info!(
+                log,
+                "Ignition response for {} ({:?}): {:?}",
+                component_id,
+                command,
+                res
+            );
```

Comment:
It seems like if we had the three-pane setup we talked about we could probably show the status in the third pane. Otherwise, the other option I can think of is us putting a spinner in the popup, keeping it open until the operation succeeds. If the operation fails, we make the popup bigger and show a message in it.